lib/appliance_console.rb

```diff
@@ -171,6 +171,10 @@ module ApplianceConsole
 
     clear_screen
 
+    # Condition the line so Backspace clears the character on screen. Setting provide equivelent line
```

Comment:
typo: equivalent

Also, the grammar feels awkward, making it a little tough to read at first.  Maybe "The setting provides the equivalent line settings when the console is run via ..." ?